src/Compilers/Core/Portable/SourceGeneration/GeneratorDriver.cs

```diff
@@ -249,11 +251,14 @@ internal GeneratorDriverState RunGeneratorsCore(Compilation compilation, Diagnos
                 using var generatorTimer = CodeAnalysisEventSource.Log.CreateSingleGeneratorRunTimer(state.Generators[i]);
                 try
                 {
-                    var context = UpdateOutputs(generatorState.OutputNodes, IncrementalGeneratorOutputKind.Source | IncrementalGeneratorOutputKind.Implementation, cancellationToken, driverStateBuilder);
-                    (var sources, var generatorDiagnostics) = context.ToImmutableAndFree();
+                    // We do not support incremental step tracking for v1 generators, as the pipeline is implicitly defined.
+                    // TODO: Re-enable this for testability
```

Comment:
>// TODO: Re-enable this for testability [](http://example.com/codeflow?start=20&length=39)

Is there work needed here? If so, it should probably be captured in an issue; otherwise, let's remove the TODO. Similar comment for other TODOs in this change.